### ISSUE TITLE
[Errno2] No Such File or Directory

### ISSUE THREAD

**Issue by drastiqJ** [patsytau/ce_tools#2]
So after fixing Cry's 5.3 solution problem(s), I finally got to the point in which I could use this script to compile/package everything down. 

However, on starting it I receive the following error:

```
Traceback (most recent call last):
  File "release_ce_project.py", line 153, in <module>
    main()
  File "release_ce_project.py", line 146, in main
    package_assets(project_path, export_path)
  File "release_ce_project.py", line 92, in package_assets
    shutil.copyfile(itempath, os.path.join(export_path, 'Assets', itemname))
  File "D:\Users\drast\AppData\Local\Programs\Python\Python35\lib\shutil.py", line 115, in copyfile
    with open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\drast\\Desktop\\ce_game\\Assets\\animations.cryasset.pak'
```
It appears it is having issues accessing, moving, or creating the folder/placing the file within it. 
I've checked all the permissions, and rights. All seem well. 

I'm using the vanilla script, with a few changes to the code's pathing/file structure:

```
def main():
    engine_version = '5.3'
    engine_path = os.path.join("D:", os.sep, "Program Files (x86)", "Crytek", "CRYENGINE Launcher", "Crytek", "CRYENGINE_5.3").format(engine_version)
    
    # Path to the project as created by the launcher.
    # project_path = os.path.join(engine_path, 'Templates', 'cpp', 'RollingBall')
    project_path = os.path.join("D:", os.sep, "Cryengine Projects", "GameSDK")
```

**Comment by patsytau** [comment]
Hiya, I'll take a look into this now.

**Comment by patsytau** [comment]
This should now be fixed by commit 01eb28385561d54c18c2f6bf53b5223fc6c79b5e, please let me know if this isn't the case for you.
